pkg/api/errors/pubsub.go

```diff
@@ -0,0 +1,204 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+import (
+	"fmt"
+	"net/http"
+
+	grpcCodes "google.golang.org/grpc/codes"
+
+	kiterrors "github.com/dapr/kit/errors"
+)
+
+const (
+	PostFixNameEmpty = "NAME_EMPTY"
+)
+
+func PubSubNotFound(name string, pubsubType string, metadata map[string]string) error {
```

Comment:
```suggestion
func NotFound(component, name string, type string, metadata map[string]string) error {
```

If we use something like this, we can reduce many duplicate codes.

Reply:
Agree